<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vanityGui</class>
 <widget class="QMainWindow" name="vanityGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vanityGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="addressInputLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Design your address here:</string>
    </property>
   </widget>
   <widget class="QLabel" name="optionsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="useCpuCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Use CPU</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="useGpuCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Use GPU</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="limitCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number of addresses to generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="advancedOptionsLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Advanced Options (for experienced users)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="threadsCpuSpinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="threadsCpuCheckBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPU threads</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="deviceGpuCheckBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GPU device</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="deviceGpuSpinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="platformGpuCheckBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GPU platform</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="platformGpuSpinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="workSizeGpuCheckBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GPU local work size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="workSizeGpuLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start!</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeEstimatesLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Time Estimate (Indicative only, car vary a lot)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="attemptsLeftLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalAttempsLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="limitSpinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="progressBarLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Estimated progress</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="generateSeedCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate seed</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressResultLineEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>nano_3sendnudez9x8zkaid5k17g474f9dc4fuopggzojwop9sbc5isgwah5gbx9n</string>
    </property>
    <property name="cursorPosition">
     <number>1</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>nano_************************************************************</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="seedLineEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BCCCCB4C0C0AC9992284B042338196CCC36CE56C4BCE2253293AE71ED56757CC</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="publicResultLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Public Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="seedLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Seed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalAttempsLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Total attempts:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="attempsLeftLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Attempts left:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLeftLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Time left:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeLeftLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="oneThreeComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>any</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="prefixLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>prefix</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="suffixLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>suffix</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nano_PrefixLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>nano_</string>
    </property>
   </widget>
   <widget class="QLabel" name="adressSampleLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>You address could look like this:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressSampleLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>nano_3sendnudez9x8zkaid5k17g474f9dc4fuopggzojwop9sbc5isgwah5gbx9n</string>
    </property>
    <property name="cursorPosition">
     <number>1</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>nano_************************************************************</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNano_Vanity">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuNano_Vanity"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
